jupyterhub/scopes.py

```diff
@@ -76,20 +79,21 @@
     'admin:server_state': {'description': 'Read and write users’ server state.'},
     'servers': {
         'description': 'Start and stop user servers.',
-        'subscopes': ['read:servers'],
+        'subscopes': ['read:servers', 'delete:servers'],
     },
     'read:servers': {
         'description': 'Read users’ names and their server models (excluding the server state).',
         'subscopes': ['read:users:name'],
     },
+    'delete:servers': {'description': "Stop users' servers."},
```

Comment:
How does this relate to named servers? I believe a named server can be both stop/started, but also deleted.

Reply:
They are not treated separately at the moment. While technically it's a separate operation, it's very rarely (if ever) treated separately, so I'd like to wait for someone to request this feature. I've updated the description to include "and delete"

Reply:
> They are not treated separately at the moment. While technically it's a separate operation, it's very rarely (if ever) treated separately, so I'd like to wait for someone to request this feature.

:+1: 

> I've updated the description to include "and delete"

Great! Note that this change is not yet pushed though - right?

Reply:
Good catch! Switching between too many branches today. Pushed now.